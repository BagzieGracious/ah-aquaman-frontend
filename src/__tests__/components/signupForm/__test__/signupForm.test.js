import { shallow } from "enzyme";
import React from "react";
import Signup from "../../../../components/signupForm";

describe("signupform on success registration", () => {
  const props = {
    onSubmit: jest.fn(),
    errors: null,
    onClick: jest.fn(),
  };
  
  it("should render without crashing", () => {
    const wrapper = shallow(<Signup {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("signupform on registration failure", () => {
  const props = {
    onSubmit: jest.fn(),
    errors: {},
  };
  it("should render without crashing", () => {
    const wrapper = shallow(<Signup {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
