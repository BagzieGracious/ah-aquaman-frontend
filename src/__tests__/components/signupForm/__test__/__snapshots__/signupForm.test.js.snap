// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`signupform on registration failure should render without crashing 1`] = `ShallowWrapper {}`;

exports[`signupform on success registration should render without crashing 1`] = `ShallowWrapper {}`;
